### DISCUSSION TITLE
Scheuduled Runs

### DISCUSSION THREAD

**Discussion by justm788** [Weatherlights/Winget-AutoUpdate-Intune#17] [Q&A]
What happens if the PC is not online when the auto-update is set to run at a specific time of day?
Will the update run the next time the PC connects or will it just run when the next planned interval comes around?

**Comment by chwet** [comment]
The admx files allows you to schedule a time and during logon so you can do both to cover bases.

**Comment by justm788** [comment]
understood
and in case the device is offline does that mean that the particular scheduled run will get skipped for good?

**Comment by chwet** [comment]
It does matter if the device is offline… on next boot you can set the updates to run on startup… it’s a scheduled task so you can go in task settings and manipulate what happens with missed tasks but isn’t necessary… the point is you want regular updates 👍

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Weatherlights** [reply]
I will add an option to the ADMX template to precheck this value. It is a good idea and I recived this question now multiple times :).

**Comment by evertoncd-f** [comment]
Hello,

What day of the week the scheduled will be executed when I select the option Weekly? For example, all Wednesday? Its possible check the information?

**Reply by mos157** [reply]
Hello, 
I have the same question. Did you find an anwser? Thank you

**Comment by cu-ri-os-i-ty** [comment]
In addition to the task scheduler, it states it will trigger on an event—Log:Microsoft-Windows-DeviceManagement-Enterprise-Diagnostics-Provider. I have a computer that is not in MDM. Can you change this or have an admx that enables us to say at login? I see that you have one, but when selected, the task scheduler does not update to say that, but still shows as the Log:Microsoft. 

Thanks

**Reply by Weatherlights** [reply]
Will check it out.
